pom.xml

```diff
@@ -163,6 +170,59 @@
             <version>2.5.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.sqlserver</groupId>
+            <artifactId>mssql-jdbc</artifactId>
+            <version>8.4.1.jre11</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Why do we need licensing dependency?

Reply:
It's being used in the IT tests to get config keys for some Embedded Cluster config
`import static io.confluent.connect.utils.licensing.LicenseConfigUtil.CONFLUENT_TOPIC_REPLICATION_FACTOR_CONFIG;`
Since these are in test scope it shouldn't matter